### ISSUE TITLE
Provider does not have a package available for darwin_arm64 platform

### ISSUE THREAD

**Issue by mikluko** [OJFord/terraform-provider-mullvad#3]
```
│ Error: Incompatible provider version
│
│ Provider registry.terraform.io/ojford/mullvad v0.2.1 does not have a package available for your current platform,
│ darwin_arm64.
│
│ Provider releases are separate from Terraform CLI releases, so not all providers are available for all platforms.
│ Other versions of this provider may have different platforms supported.
╵
```

**Comment by mikluko** [comment]
Looks like it is only because of the old `goreleaser` version. With v1.9.2 I'm able to build `darwin_arm64` without any modifications to the code.

**Comment by OJFord** [comment]
Oh hello again, I thought this was familiar (OJFord/terraform-provider-wireguard#9) 😄 

Thanks, will have a look.

**Comment by OJFord** [comment]
Closed via v0.2.2, thanks for the report!

**Comment by mikluko** [comment]
> Oh hello again, I thought this was familiar (OJFord/terraform-provider-wireguard#9) 😄 
> 
> 
> 
> Thanks, will have a look.

I completely forgot about the original one 🤦‍♂️

Thanks for the prompt fix!
